tests/test_numpy_conversions.py

```diff
@@ -89,6 +104,84 @@ def test_check_ros_numpy_joint_state_conversion() -> None:
+    quat = np.random.rand(4)
+    quat = quat / np.linalg.norm(quat)
+
+    numpy_control = lfc_py_types.Control(
+        initial_state=lfc_py_types.Sensor(
+            base_pose=np.concatenate((np.random.rand(3), quat)),
+            base_twist=np.random.rand(6),
+            joint_state=lfc_py_types.JointState(
+                name=["1", "2", "3", "4", "5", "6"],
+                position=np.random.rand(6),
+                velocity=np.random.rand(6),
+                effort=np.random.rand(6),
+            ),
+            contacts=[],
+        ),
+        feedback_gain=np.random.rand(8, 4),
+        feedforward=np.random.rand(4),
+    )
+
+    ros_control_msg = control_numpy_to_msg(deepcopy(numpy_control))
+    back_converted_numpy_control = control_msg_to_numpy(ros_control_msg)
+
+    np.testing.assert_array_equal(
+        numpy_control.initial_state.base_pose,
+        back_converted_numpy_control.initial_state.base_pose,
+        err_msg="Initial state base pose after conversion back to "
+        + "Numpy is not equal initial values!",
+    )
+
+    np.testing.assert_array_equal(
+        numpy_control.initial_state.base_twist,
+        back_converted_numpy_control.initial_state.base_twist,
+        err_msg="Initial state base twist after conversion back to "
+        + "Numpy is not equal initial values!",
+    )
+
+    assert (
+        numpy_control.initial_state.joint_state.name
+        == back_converted_numpy_control.initial_state.joint_state.name
+    ), "Names after conversion back to Numpy is not equal initial values!"
+
+    np.testing.assert_array_equal(
+        numpy_control.initial_state.joint_state.position,
+        back_converted_numpy_control.initial_state.joint_state.position,
+        err_msg="Joint state position after conversion back to "
+        + "Numpy is not equal initial values!",
+    )
+
+    np.testing.assert_array_equal(
+        numpy_control.initial_state.joint_state.velocity,
+        back_converted_numpy_control.initial_state.joint_state.velocity,
+        err_msg="Joint state velocity after conversion back to "
+        + "Numpy is not equal initial values!",
+    )
+
+    np.testing.assert_array_equal(
+        numpy_control.initial_state.joint_state.effort,
+        back_converted_numpy_control.initial_state.joint_state.effort,
+        err_msg="Joint state effort after conversion back to "
+        + "Numpy is not equal initial values!",
+    )
+
+    np.testing.assert_array_equal(
+        numpy_control.feedback_gain,
+        back_converted_numpy_control.feedback_gain,
+        err_msg="Feedback gains after conversion back to "
+        + "Numpy is not equal initial values!",
+    )
+
+    np.testing.assert_array_equal(
+        numpy_control.feedforward,
+        back_converted_numpy_control.feedforward,
+        err_msg="Feed forward after conversion back to "
+        + "Numpy is not equal initial values!",
+    )
+
+
 def test_check_ros_eigen_sensor_conversions() -> None:
```

Comment:
```suggestion
def test_check_ros_numpy_sensor_conversions() -> None:
```
Please check all the test function names replacing eigen into numpy

Reply:
Fixed